### PR TITLE
Issue-1480 Add npm adduser to NPM supported commands

### PR DESCRIPTION
**Pull request by LeifBrooks**
This updates the docs for strongbox/strongbox#1480.

### CODE REVIEW

File: docs/developer-guide/layout-providers/npm-layout-provider.md [hunk 1]
```diff
@@ -29,7 +29,9 @@ Following [CLI commands](https://docs.npmjs.com/cli/npm) supported:
 
 - `npm view` 
 
-- `npm search` 
+- `npm search`
+
+- `npm adduser`
```
Comment (carlspring) [inline docs/developer-guide/layout-providers/npm-layout-provider.md]: If you've tested it with both of these, perhaps you could change the line to this:

```suggestion
- `npm [adduser|login]` 
```

...?

Comment (LeifBrooks) [reply docs/developer-guide/layout-providers/npm-layout-provider.md]: Updated the text

**Comment by carlspring** [comment]
Thanks! It looks good! :)

I'll merge this once the other pull request has been reviewed and approved.

**Comment by carlspring** [comment]
Thanks! :)

**Pull request merged**